src/utils/searchUtils.ts

```diff
@@ -17,9 +17,18 @@ import { index } from "../data/index.gen";
 // ───────────────────────────────────────────────────────────────────────────────
 export function getGlobalFacetOptions(pathways: PathwayMetadataType[]) {
   // Pathway Type
-  const pathwayTypeOptions = buildOptionsFromValues(
-    pathways.map((d) => d.pathwayType),
-  );
+  // Reorder pathwayType options: Predictive, Exploratory, Normative, Direct Policy
+  const rawPathwayTypes = pathways.map((d) => d.pathwayType);
+  const orderedTypes = [
+    "Predictive",
+    "Exploratory",
+    "Normative",
+    "Direct Policy",
+  ];
+  const pathwayTypeOptionsUnsorted = buildOptionsFromValues(rawPathwayTypes);
+  const pathwayTypeOptions = orderedTypes
+    .map((type) => pathwayTypeOptionsUnsorted.find((opt) => opt.label === type))
+    .filter(Boolean);
```

Comment:
I generally prefer the construction in the other file, and if you extract that to a function, then I think that this can become something like

```suggestion

  const pathwayTypeOptions = buildOptionsFromValues(
    pathways.map((d) => d.pathwayType),
  )
  .sortPathwayType();
```